cupy_setup_build.py

```diff
@@ -106,19 +109,6 @@
         'check_method': build.check_nccl_version,
         'version_method': build.get_nccl_version,
     },
-    {
-        'name': 'cusolver',
```

Comment:
Should these code be fixed too? https://github.com/cupy/cupy/blob/93b656201c4d032b26caa52220e6bf5f4470fc6f/cupy_setup_build.py#L389-L398